### ISSUE TITLE
Adapter 1.7.0: missing datapoint

### ISSUE THREAD

**Issue by JarlMorgennebel** [ioBroker/ioBroker.wolf#96]
Hej,

```
wolf.0 | 2023-01-29 17:05:39.169 | warn | State "wolf.0.bm1_t.194" has no existing object, this might lead to an error in future versions
-- | -- | -- | --
wolf.0 | 2023-01-29 16:33:49.940 | warn | State "wolf.0.bm1_t.194" has no existing object, this might lead to an error in future versions
wolf.0 | 2023-01-29 16:02:31.063 | warn | State "wolf.0.bm1_t.194" has no existing object, this might lead to an error in future versions
wolf.0 | 2023-01-29 15:31:32.069 | warn | State "wolf.0.bm1_t.194" has no existing object, this might lead to an error in future versions
```

Adapter 1.7.0.

Thanks, JMN

**Comment by Claus1985** [comment]
got the same error also on v1.7:
`State "wolf.0.bm1_t.194" has no existing object, this might lead to an error in future versions`

Thx for the great work btw!!

**Comment by tobi119** [comment]
Gleicher Fehler bei mir nach Update von Version 1.50 auf V1.70
Bei 1.50 ist der DP im Objektbaum an einer anderen Stelle.
Nach Löschen des DP und Neustart des Adapters wurde DP neu angelegt. Fehler tritt nicht mehr auf.

Habe jetzt aber einen anderen Fehler:
info	For datapoint 362 no device defined.
info	For datapoint 363 no device defined.

Steht alle 30 Min je 6 mal im Log

`2023-02-06 16:11:17.715	info	For datapoint 363 no device defined.
2023-02-06 16:11:17.712	info	For datapoint 362 no device defined.
2023-02-06 16:11:17.615	info	For datapoint 363 no device defined.
2023-02-06 16:11:17.611	info	For datapoint 362 no device defined.
2023-02-06 16:11:17.512	info	For datapoint 363 no device defined.
2023-02-06 16:11:17.508	info	For datapoint 362 no device defined.
2023-02-06 16:11:17.409	info	For datapoint 363 no device defined.
2023-02-06 16:11:17.407	info	For datapoint 362 no device defined.
2023-02-06 16:11:17.309	info	For datapoint 363 no device defined.
2023-02-06 16:11:17.305	info	For datapoint 362 no device defined.
2023-02-06 16:11:17.207	info	For datapoint 363 no device defined.
2023-02-06 16:11:17.205	info	For datapoint 362 no device defined.
2023-02-06 16:10:58.479	info	Invalid polling interval. Disable polling.
2023-02-06 16:10:58.443	info	starting. Version 1.7.0 in /opt/iobroker/node_modules/iobroker.wolf, node: v16.17.0, js-controller: 4.0.23
2023-02-06 16:10:56.489	info	instance system.adapter.wolf.0 started with pid 6964
`

**Comment by Apollon77** [comment]
it is correct the 363 and 362 are not yet defined. Do you know the definition? @fred0815 any idea maybe?

**Comment by fred0815** [comment]
Yes it is correct. Both are defined by Wolf as internal DPs. I ignored them cause i got only zeros the hole time.

**Comment by Apollon77** [comment]
Maybe we should simply add themm also when it is just 0?  ;-)

**Comment by fred0815** [comment]
I have added both DPs. 362 and 363 
From firmware 1.80 and above there are two more DPs 372 and 767 (no idear of definition)
Due to documentation I have modified the device and DPs mappings
    if ((dp >= 1 && dp <= 13) || (dp >= 176 && dp <= 191) || (dp >= 197 && dp <= 199) || (dp === 364)) {
        return 'hg1_t';
    } else if ((dp >= 14 && dp <= 26) || (dp >= 200 && dp <= 202) || (dp === 365)) {
        return 'hg2_t';
    } else if ((dp >= 27 && dp <= 39) || (dp >= 203 && dp <= 205) || (dp === 366)) {
        return 'hg3_t';
    } else if ((dp >= 40 && dp <= 52) || (dp >= 206 && dp <= 208) || (dp === 367)) {

hg0_t does not exist anymore 
May be @Apollon77 you can delete the "Comfort domestic ventilation system" section in the "index_m.html" file
[main.js.txt](https://github.com/ioBroker/ioBroker.wolf/files/10686125/main.js.txt)
[datapoints.json.txt](https://github.com/ioBroker/ioBroker.wolf/files/10686126/datapoints.json.txt)
[index_m.html.txt](https://github.com/ioBroker/ioBroker.wolf/files/10686127/index_m.html.txt)

**Comment by fred0815** [comment]
In addition I found some inconsistencies regarding "boolean" values.

while creating the objects for Iobroker I have added "DPT_Bool" to the "if" statement

**if (adapter.config.bool_status && ['DPT_Switch', 'DPT_Enable', 'DPT_OpenClose', 'DPT_Bool'].includes(data.type)) {
						_data.commonType = 'boolean';_**

corrected some "'true'" and false values to "true" and "false" (without inverted commas) to be "real" boolen values..

[main.js.txt](https://github.com/ioBroker/ioBroker.wolf/files/10694946/main.js.txt)

May be @Apollon77 you can add some "verifying code" if someone changed the "status als Boolean" in the Wolf adapter instances view after all objects have been created in Iobroker already. Otherwise you will get tons of "Infos" of incorrent Type Informations in the log file...

**Comment by fred0815** [comment]
@Apollon77 any propose to update the adapter?

**Comment by Apollon77** [comment]
I have one milestone left on my "Matter list" for now, then I do an other Adapter break and check that

**Comment by franks06347** [comment]
Fehler tritt bei mir auch auf:
[iobroker.Wolf._LOG.2023-11-06.log](https://github.com/ioBroker/ioBroker.wolf/files/13266339/iobroker.Wolf._LOG.2023-11-06.log)

**Comment by franks06347** [comment]
Nabend,
ich habe die beiden Dateien runtergeladen und ersetzt
from issue https://github.com/ioBroker/ioBroker.wolf/issues/96
"datapoints,json" to "/opt/iobroker/node_modules/iobroker.wolf/js"
"main.js" to "/opt/iobroker/node_modules/iobroker.wolf"

leider immer noch viele Fehler:

[wolf_log_20231106_1830.log](https://github.com/ioBroker/ioBroker.wolf/files/13270616/wolf_log_20231106_1830.log)

**Comment by fred0815** [comment]
Ich prüfe es nochmal..
